src/main/java/org/sagebionetworks/web/client/widget/team/controller/TeamEditModalWidget.java

```diff
@@ -159,18 +159,23 @@ private void show() {
     view.clear();
     view.configure(team);
     //SWC-6259
-    Boolean canPublicJoin = team.getCanPublicJoin();
+    //SWC-7000
+    Boolean canPublicJoin = team.getCanPublicJoin() == null
```

Comment:
These should never be null since https://sagebionetworks.jira.com/browse/PLFM-8304 was done